packages/data-service/src/data-service.ts

```diff
@@ -2645,6 +2695,31 @@ class DataServiceImpl extends WithLogContext implements DataService {
     };
   }
 
+  async listStreamProcessors(filter?: Document): Promise<StreamProcessor[]> {
+    const { streamProcessors } = await this._runStreamCmd({
+      listStreamProcessors: 1,
+      filter,
+    });
+    return streamProcessors;
+  }
+
+  async startStreamProcessor(name: string): Promise<void> {
```

Comment:
> Do the new methods need to annotated with @op and what would be the logId for them if so?

Yes, all methods that run driver commands should have an `op` decorator applied so that the logging and errors are normalized. The id is the next available one as returned by `npm run check-logids` or using eslint fix suggestions if you have integration implemented.

Reply:
Have annotated with `@op`.